build/package/Layout.targets

```diff
@@ -30,9 +30,10 @@
       <Output TaskParameter="RelativePath" ItemName="SdkRelativeOutputFiles" />
     </MakeRelative>
     <MakeRelative
-      Path1="$(SymbolsDirectory)"
-      Path2="%(SdkDebugLayoutInput.Identity)" >
-      <Output TaskParameter="RelativePath" ItemName="SdkDebugRelativeOutputFiles" />
+      Condition="'$(SeparateSymbolsArchive)' != 'false'"
```

Comment:
When is this set?

Reply:
See description, it's an option for local debugging/profiling

Reply:
Got it.